Gemfile

```diff
@@ -1,25 +1,26 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '2.5.3'
+ruby '3.1.2'
 
-gem 'autoprefixer-rails'
-gem 'bootsnap', '>= 1.4.2', require: false
-gem 'bootstrap-sass', '~> 3.4.1'
-gem 'devise'
-gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'master'
+gem 'autoprefixer-rails' 
+gem 'bootsnap', '>= 1.4.2', require: false 
+gem 'bootstrap-sass', '~> 3.4.1' 
+gem 'devise' 
+gem 'faker', :git => 'https://github.com/stympy/faker.git', :branch => 'main'
```

Comment:
Can you make sure that line 6-10 remain the same?

Reply:
The 'faker' repo has been changed so that 'main' has replaced 'master'  -- bundle fails w/o this change.

Reply:
Ok, I was thinking that was in reference to our `main` branch. That makes more sense.